src/main/java/bookrepo/dto/CreateBookRequestDto.java

```diff
@@ -1,16 +1,25 @@
 package bookrepo.dto;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
for String validation use NotBlank
```suggestion
    @NotBlank
```